src/instruments/src/EFB/Navigation/Pages/LocalFilesPage/LocalFilesPage.tsx

```diff
@@ -12,28 +15,32 @@ enum ConnectionState {
 }
 
 export const getPdfUrl = async (fileName: string, pageNumber: number): Promise<string> => {
+    const simbridgeEnabled = NXDataStore.get('CONFIG_SIMBRIDGE_ENABLED', 'AUTO ON');
+    if (simbridgeEnabled !== 'AUTO ON') {
+        toast.error('SimBridge is not enabled in flyPad settings.');
+        return Promise.reject();
+    }
     try {
-        const resp = await fetch(`http://localhost:8380/api/v1/utility/pdf?filename=${fileName}&pagenumber=${pageNumber}`);
-
-        if (!resp.ok) {
-            toast.error('Failed to retrieve requested PDF Document.');
-            return Promise.reject();
-        }
-
-        const blob = await resp.blob();
+        const blob = await Viewer.getPDFPage(fileName, pageNumber);
```

Comment:
Just a reminder here to have this return the objectURL string rather than a blob.

Reply:
Same question as above - you want me to change the SimBridge Client API?

Reply:
done